### DISCUSSION TITLE
About github offers

### DISCUSSION THREAD

**Discussion by 24sa12** [github/india#846] [GitHub Platform]
As I'm new to github services which I got from student developer pack, can you please give a step by step procedure that how can I avail all these offers.
Means how can I make use of it ?

**Comment by prathikshetty2002** [comment]
GitHub student developer pack provides you with a lot of benefits. Like 3-6 month free subscription to learning platforms like data camp and frontend masters. Tool credits like digital ocean and azure etc. Now if you want to avail them , you need to connect your GitHub account with your service that you want to avail. 

If you are not sure about it go to https://education.github.com/pack and learn more

Hope it helps, have a nice day

**Comment by FRTMentor** [comment]
Hi @24sa12 ,
https://www.youtube.com/watch?v=GuuNbzmve8Q
https://www.youtube.com/watch?v=A8U3FXUcR5k
https://www.youtube.com/watch?v=V4WtypyR9vE
Please reach out our support team.
support@futurereadytalent.in
